libs/ui-lib/lib/ocm/components/clusterWizard/ClusterDetailsForm.tsx

```diff
@@ -175,6 +175,7 @@ const ClusterDetailsForm = (props: ClusterDetailsFormProps) => {
                   clusterExists={!!cluster}
                   clusterCpuArchitecture={cluster?.cpuArchitecture as CpuArchitecture}
                   clusterId={cluster?.id}
+                  clusterPlatform={cluster?.platform?.type}
```

Comment:
Add a default value of `none` when there is no cluster as that's the correct default, and as mentioned earlier, will make the components more robusts as they will always receive a `PlatformType`

Reply:
Changed it